internal/app/command_test.go

```diff
@@ -5,6 +5,44 @@ import (
 	"testing"
 )
 
+func Test_toolExists(t *testing.T) {
+	type args struct {
+		tool string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "test case 1 -- checking helm exists.",
+			args: args{
+				tool: helmBin,
+			},
+			want: true,
+		}, {
+			name: "test case 2 -- checking kubectl exists.",
```

Comment:
It's weird to mandate `kubectl` and `helm` are installed...this means these tests can't predictably be run locally, on any machine.

Reply:
yeah but these tests already existed I've just moved them to a file where I think they make more sense since the `toolExists` method is implemented in `utils.go` and not `cli.go`

Reply:
Sure, but if you agree we could probably just change the check to `ls` and be done with it. 😄

Reply:
sure but I think we have tests that actually depend on those tools...

Reply:
Oof, fair enough. 😃

Reply:
It's not a big deal, but maybe some day we can get rid of this.